spec/scry/completion_provider_spec.cr

```diff
@@ -128,9 +128,20 @@ module Scry
 
         it_completes "Node.", %w(new)
 
-        it_completes "a = A.new
+        it_completes "a = B.new
                       a.", %w(method)
       end
     end
+
+    context "classname completion" do
```

Comment:
Note: it can also complete module names (and structs), not only class names, maybe change the spec name?